src/modules/private.ts

```diff
@@ -587,6 +590,57 @@ export default class Private {
     );
   }
 
+  /**
+     * @description post a new transfer
+     *
+     * @param {
+      * @amount specifies the size of the transfer
+      * @receiverAccountId specifies the receiver account id
+      * @receiverPublicKey specifies the receiver public key
+      * @receiverPositionId specifies the receiver position id
+      * @clientId specifies the clientId for the address
+      * @signature starkware specific signature for the transfer
+      * }
+      */
+  async createTransfer(
+    params: PartialBy<TransferParams, 'clientId' | 'signature'>,
+    positionId: string,
+  ): Promise<{ transfer: TransferResponseObject }> {
+    const clientId = params.clientId || Math.random().toString().slice(2).replace(/^0+/, '');
```

Comment:
nit: `Math.random().toString().slice(2).replace(/^0+/, '')` to a helper function `generateRandomClientId()`? since it is used in a few different functions?

Reply:
added